### PR TITLE
executor: support Chunk for ShowDDLJobsExec

### PR DESCRIPTION
**Pull request by zz-jason**
to #5261

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -255,6 +275,21 @@ func (e *ShowDDLJobsExec) Next(goCtx goctx.Context) (Row, error) {
 	return row, nil
 }
 
+// NextChunk implements the Executor NextChunk interface.
+func (e *ShowDDLJobsExec) NextChunk(goCtx goctx.Context, chk *chunk.Chunk) error {
+	chk.Reset()
+	if e.cursor > len(e.jobs) {
```
Comment (coocood) [inline executor/executor.go]: >=

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**